database/rrdset.c

```diff
@@ -1932,6 +1932,18 @@ void rrdset_finalize_labels(RRDSET *st)
     } else {
         replace_label_list(labels, new_labels);
     }
+#ifdef ENABLE_DBENGINE
+    if (likely(st->rrd_memory_mode == RRD_MEMORY_MODE_DBENGINE)) {
```

Comment:
thinking out loud... no sure `likely` is appropriate here. For people using DBENGINE, modern branch prediction will take the hint even without this. For people who use another memory mode, this will actually make things worse

Reply:
No problem, I will remove it (besides the next PR down the road will remove this all together. It will work in any memory mode.

Reply:
ah ok in such case no need to remove it from this PR ... up to you